crates/cargo-test-support/src/compare.rs

```diff
@@ -41,6 +41,57 @@ use std::path::Path;
 use std::str;
 use url::Url;
 
+/// Default `snapbox` Assertions
+///
+/// # Snapshots
+///
+/// Updating of snapshots is controlled with the `SNAPSHOTS` environment variable:
+///
+/// - `skip`: do not run the tests
+/// - `ignore`: run the tests but ignore their failure
+/// - `verify`: run the tests
+/// - `overwrite`: update the snapshots based on the output of the tests
+///
+/// # Patterns
+///
+/// - `[..]` is a character wildcard, stopping at lin breaks
+/// - `\n...\n` is a line wildcard
```

Comment:
It's not clear to me, why would one use this instead of `[..]`?

Reply:
``\n...\n` is a line wildcard` was clarified to ``\n...\n` is a multi-line wildcard`

A line with just `[..]` will only match that one line.  `\n...\n` will match multiple lines.  This is helpful if you don't know how many lines you need elided (e.g. it changes based on the platform)